packages/lesswrong/lib/collections/posts/views.ts

```diff
@@ -262,6 +268,32 @@ const getFrontpageFilter = (filterSettings: FilterSettings): {filter: any, softF
   }
 }
 
+function buildInflationAdjustedField(): any {
+  const karmaInflationSeries = getKarmaInflationSeries();
+  return {
+    karmaInflationAdjustedScore: {
+      $multiply: [
+        "$baseScore",
+        {
+          $ifNull: [
+            {
+              $arrayElemAt: [
+                karmaInflationSeries.values,
+                {
+                  $max: [
+                    timeSeriesIndexExpr("$postedAt", karmaInflationSeries.start, karmaInflationSeries.interval),
+                    0 // fall back to first value if out of range
+                  ]
+                }]
+            },
```

Comment:
this is effectively using `baseScore / meanKarmaForMonth` as the field to sort by